sycl/test/regression/sycl_api.cpp

```diff
@@ -0,0 +1,176 @@
+// RUN: %clang -std=c++11 -fsycl %s -o %t.out -lstdc++ -lOpenCL
```

Comment:
```suggestion
// RUN: %clangxx -fsycl %s -o %t.out -lOpenCL
```
This should enable this test for Windows.

Reply:
thanks